monero-outreach-docs/en/quickfacts/quickfacts.md

```diff
@@ -1,12 +1,12 @@
 # MONERO
 
-Monero is a secure, private, untraceable digital currency that is available to all.
+Monero is a secure, private, untraceable digital currency that is available to all. It’s features make it fungible cash.
 
 # QUICK FACTS
 
-Users can spend safely, no one can see their balances or track their activity.
+Users can spend safely, no one can see their balances or track their activity unless the user gives them permission.
 
-Revised 11/14/2018 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)    
+Revised 2/1/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf) 
 Created for the community by Monero Outreach
 
 - [Full Color Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts.pdf)
```

Comment:
I think this should be a 2019 folder too.

Reply:
Same as last comment, we might do it for the next update so we won't mess everything right now.

Reply:
ok